README.md

```diff
@@ -59,6 +59,7 @@ var crop = Crop.create({
     x: 100,
     y: 500
   },
+  ignoreDevicePixelRatio: false,
```

Comment:
Good idea, but probably better to accept explicit `devicePixelRatio` value i.e. in your case, just set `devicePixelRatio: 1` - does that sound okay?

Reply:
I've tried that and it works, but I was afraid that changing _devicePixelRatio_ could make problem somewhere else in app.

I was making some crop area with Vue and using _ignoreDevicePixelRatio_ it's responsive now :). 

Very nice interface, thank you.